ckanext/canada/templates/internal/user/login.html

```diff
@@ -52,7 +52,7 @@
     {% endif %}
 
     {% block form %}
-      {% snippet 'user/login_form.html', action=c.login_handler, error_summary=error_summary %}
+      {% snippet 'user/snippets/login_form.html', action=c.login_handler, error_summary=error_summary %}
```

Comment:
you also changed the variable passed to error_summary. Is the original block in ckanext-wet-boew incorrect? If so fix it there, if not, you don't need to override this block.